assets/styles.css

```diff
@@ -0,0 +1,177 @@
+body {
+  overflow-x: hidden;
+}
+
+.body-wrapper {
+  width: 1210px;
+  margin: 0 auto;
+}
+
+.header-wrapper {
+  height: 100px;
+  color: white;
+  background-color: #f72210;
+}
+
+.font-head a {
+  font-size: 14px;
+}
+
+.logo-head {
+  color: #fff;
+  fill: white;
+  width: 400px;
+}
+
+.header-font {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.rounded-pill {
+  background-color: #ffe400;
+  height: 30px;
+}
+
+.search-input {
+  height: 38px;
+  width: max-content;
+}
+
+.article-body {
+  padding-top: 10px;
+  margin-bottom: 15px;
+}
+
+.main-section {
+  padding: 25px;
+}
+
+.navbar-one a {
+  padding: 15px;
+}
+
+.navbar-one li {
+  border-right: 1px solid #dee2e6 !important;
+}
+
+.newsletter-signup {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+
+.icon-signup {
+  flex: none;
+  display: block;
```

Comment:
- [ ] use `d-block` Bootstrap class instead of ` display: block;`

Reply:
done